.github/workflows/app_build.yml

```diff
@@ -151,51 +163,56 @@ jobs:
     # TODO: cache the QtIFW folder too
 
     - name: Cache OpenStudio tar.gz
-      # To avoid downloading the SDK all the time, we try to cache it
+      # To avoid downloading the SDK all the time, we try to cache it. The path matches both what FindOpenStudioSDK.cmake does and what the 'Download the OpenStudio installer' used to do
       id: cacheossdk
       uses: actions/cache@v2
       with:
         path: OpenStudio-${{ env.OS_SDK_VERSION }}
         key: OpenStudio-SDK-${{ runner.os }}-${{ env.OS_SDK_VERSION }}
 
-    - name: Download the OpenStudio installer
-      # If the SDK wasn't found in the cache
-      if: steps.cacheossdk.outputs.cache-hit != 'true'
-      # Use a bash shell so we can use the same syntax for environment variable
-      # access regardless of the host operating system
-      # Note: wget not available on windows, even in bash, so use curl
-      shell: bash
-      run: |
-        curl --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' https://api.github.com/repos/NREL/OpenStudio/releases/tags/v$OS_SDK_VERSION > os_tag.json
-        echo "PLATFORM_NAME=${{ env.PLATFORM_NAME }}"
-        cat os_tag.json
-        tar_gz_link=$(cat os_tag.json | jq -r '.assets | .[] | select(.name | contains("${{ env.PLATFORM_NAME }}")) | select(.name | contains("tar.gz")) | .browser_download_url')
-        if [ -z "$tar_gz_link" ]; then
-          echo "Could not locate the OpenStudio tar.gz for OS_SDK_VERSION=$OS_SDK_VERSION and PLATFORM_NAME=$PLATFORM_NAME"
-          exit 1;
-        fi;
-        tar_gz_name=$(basename -- $tar_gz_link)
-        tar_gz_name=$(python -c "import sys, urllib.parse as ul; print(ul.unquote_plus(\"$tar_gz_name\"))")
-        echo "Downloading $tar_gz_link"
-        echo "Normally, tar_gz_name=$tar_gz_name"
-        curl -L --output "$tar_gz_name" "$tar_gz_link"
-        ls
-        ls -la $tar_gz_name
-        echo "Extract this into the OpenStudio-$OS_SDK_VERSION directory, omitting the first directory level in the tar.gz"
-        echo "This isn't working, so use cmake to tar, then rename"
-        echo "tar xfz $tar_gz_name --strip-components 1 -C OpenStudio-$OS_SDK_VERSION"
-        echo "cmake -E make_directory OpenStudio-$OS_SDK_VERSION"
-        folder_name=${tar_gz_name%.tar.gz}
-        echo "folder_name=$folder_name"
-        cmake -E tar xfz "$tar_gz_name"
-        cmake -E rename "$folder_name" "OpenStudio-$OS_SDK_VERSION"
-
+     # NOTE JM 2020-09-01: we're going to rely on FindOpenStudioSDK.cmake for now, but it's not impossible we'll need this in the future
+     # This step in conjuction with the step 'Create Build Environment and locate openstudio', and a small modification in 'Configure CMake' step
```

Comment:
Now using `FindOpenStudioSDK.cmake` but I'm not ready to delete it from the workflow file for fear of having trouble finding it later.